configuration/pih/htmlforms/section-dx-obgyn.xml

```diff
@@ -0,0 +1,307 @@
+<htmlform formName="Diagnostic with obgyn" formVersion="1.0"
```

Comment:
Why is this form being added in this PR?  I thought this was just about adding GPAL calculation?

Reply:
@mseaton , this form came from this PR  https://github.com/PIH/openmrs-config-pihemr/pull/520/files  that @lnball created.